create_incident_example.py

```diff
@@ -13,17 +13,24 @@
 import argparse
 import demisto
 
+
 def options_handler():
-    parser = argparse.ArgumentParser(description='Utility for batch action on incidents')
-    parser.add_argument('-k', '--key', help='The API key to access the server', required=True)
-    parser.add_argument('-s', '--server', help='The server URL to connect to', required=True)
+    parser = argparse.ArgumentParser(
+        description='Utility for batch action on incidents')
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-s', '--server', help='The server URL to connect to', required=True)
```

Comment:
> Note: Required options are generally considered bad form because users expect options to be optional, and thus they should be avoided when possible.

* https://docs.python.org/3/library/argparse.html#required

Maybe these two should be positional arguments instead?

```python
parser.add_argument('key', help='The API key to access the server')
parser.add_argument('server', help='The server URL to connect to')

```

Reply:
used positional arguments instead of required=True.(e2beb08)